packages/neptune-extension/src/labextension/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { DisposableDelegate } from '@phosphor/disposable';
```

Comment:
You've removed `phosphor` and this is not gonna build properly.

Reply:
I had it offline, forgot to add it to changelist.